harness/determined/cli/model.py

```diff
@@ -122,21 +123,25 @@ def list_versions(args: Namespace) -> None:
             "Metadata",
         ]
 
-        values = [
-            [
-                version.model_version,
-                version.checkpoint.training.trial_id if version.checkpoint.training else None,
-                version.checkpoint.metadata["steps_completed"],
-                version.checkpoint.uuid,
-                (
-                    json.dumps(version.checkpoint.training.validation_metrics, indent=2)
-                    if version.checkpoint.training
-                    else ""
-                ),
-                json.dumps(version.checkpoint.metadata, indent=2),
-            ]
-            for version in model.get_versions()
-        ]
+        values = []
```

Comment:
This is better (thank you), but it still sucks to read. With the signature `list_versions` has, it seems to me like it should have an implementation like this:

```python
def list_versions(args: Namespace) -> None:
  model = get_model(...)
  versions = model.get_versions()
  render(model, versions, args)
```

and oh my goodness the more I read this module the angrier I get. Maybe just forget this comment and we'll someday refactor the CLI.

If it's all this gross, maybe we should have a policy of touching the CLI as little as possible? I dunno -- what do you think?